README.md

```diff
@@ -121,16 +136,16 @@ Below is a comprehensive list showing all zbox commands along with their respect
+| get-diff          | [Get differences](#get-differences): Get differences between the local files specified by `localpath` and the files stored on the root remotepath of the allocation.<br /><br />[Get differences with excludepath](#get-differences): Get differences between local directory and root remotepath and exclude a specific remotepath. | Get differences:`./zbox get-diff --allocation $ALLOCATION_ID --localpath <path_to_local_directory>`<br /><br />Get differences with excludepath:`./zbox get-diff --allocation $ALLOCATION_ID --localpath /home/zus/files --excludepath /myfiles` |
 | get-wallet        | [Get wallet](#get-wallet): Get wallet information.           | Get wallet: `./zbox getwallet`                               |
 | rename            | [Rename](#rename): Rename an existing file on allocation.    | Rename:`./zbox rename --remotepath /sync.txt --destname <new_name_for_the_file> --allocation $ALLOCATION_ID` |
 | stats             | [Stats](#stats): Get Stats for a file such as upload, download and challenge information for a file. | Stats:`./zbox stats --remotepath <remote_path_of_file> --allocation $ALLOCATION_ID ` |
-| get-download-cost | [Download cost](#download-cost): Get Download cost for a file on an allocation.<br /><br />[Download cost via authticket](#download-cost): Get Download cost for a shared file. | Get Download Cost :`./zbox get-download-cost --allocation $ALLOCATION_ID --remotepath <path to_remote_file>`<br /><br /><br />Get Download Cost via authticket :`./zbox get-download-cost --authticket $AUTH_TICKET --allocation $ALLOCATION_ID` |
+| get-download-cost | [Download cost](#download-cost): Get Download cost for a file on an allocation.<br /><br />[Download cost via authticket](#download-cost): Get Download cost for a shared file via authticket. | Get Download Cost :`./zbox get-download-cost --allocation $ALLOCATION_ID --remotepath <path to_remote_file>`<br /><br /><br />Get Download Cost via authticket :`./zbox get-download-cost --authticket $AUTH_TICKET --allocation $ALLOCATION_ID` |
```

Comment:
please fix the vertical alignment

Reply:
Please see my comment on top in terms of alignment.